### DISCUSSION TITLE
Login issue on clients

### DISCUSSION THREAD

**Discussion by AuhNijGnahz** [dani-garcia/vaultwarden#4229] [Q&A]
After update to the latest image, I have encountered login issue on all client-side applications I use, they are: win11 desktop app, chrome extension, ios app.

docker log:
2024-01-07 12:59:00.915][request][INFO] POST /identity/accounts/prelogin
[2024-01-07 12:59:00.915][response][INFO] 404 Not Found

any solutions there?

**Comment by BlackDex** [comment]
What version of Vaultwarden are you running and what are the client versions?

**Reply by KongNan** [reply]
I may be experiencing a similar problem. I'm using the service provided by vaultwarden.net, and my client version is 2023.12.1 from the Edge Store (updated 2023-12-20)

**Comment by rickywu** [comment]
I have the same problem, vaultwarden/server:alpine 
Chrome extension is the latest version

I found the reason, vaultwarden/server:alpine got images build in 2years ago, must use vaultwarden/server:1.30.1-alpine or use vaultwarden/server:latest

**Reply by BlackDex** [reply]
I'm not sure where you see that the `server:alpine` is build 2 years ago. According to Docker Hub it's 2 months ago.
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/1e2cd8c7-d4b2-45c0-b438-1fb082dac294)

If you want the latest alpine i would suggest to use `latest-alpine`, but `alpine` would work just fine too.

**Comment by rickywu** [comment]
![image](https://github.com/dani-garcia/vaultwarden/assets/442398/7ea7cfe6-35e8-425f-85e0-9a57abcc77fd)

@BlackDex see this, I reprocuded again, vaultwarden/server:latest got the same problem, I have to specify version number

**Reply by BlackDex** [reply]
Did you ever do a `docker pull vaultwarden/server:alpine` in the past two years? Or any other image? Since i see a lot of old images there. Gitea for example updated like 5 days ago, yours is 16months old.

It looks like you never update your images. Just one pull, and done.

**Reply by BlackDex** [reply]
Ow, wait, you might also be using a docker mirror which isn't updating those images, or causes other issues.

But that is not something we can fix.

**Reply by rickywu** [reply]
Do you mean docke.io is not updated?

**Reply by BlackDex** [reply]
docker.io is updated as  you can see in my screenshot, as that is coming from docker hub.
Since all other images also seem very outdated on your system, it must be something on your system which doesn't want to pull new images.

**Comment by rickywu** [comment]
I think this problem caused by old vesion, I see the same error

**Reply by BlackDex** [reply]
It is, old versions of Vaultwarden do not support the new login/sync needed by the clients.
Therefor you need to update your container as mentioned above.

**Reply by kpfleming** [reply]
And old versions of Docker may not be able to pull the latest versions of the images. What version of Docker is being used?

**Reply by BlackDex** [reply]
Ah yes, i almost forgot that.
You need at least v20.10 if i am correct.

**Reply by rickywu** [reply]
I found the reason, I'm using dsm 7.2 and need config new registry mirror to pull newest image

**Reply by Gerardv514** [reply]
I’m on dsm 7.2 and I’m working fine. I’ve never had an issue pulling the latest version and when one is released I see it without delay. 

Maybe the difference is I’m using the dsm built in container manager, whereas you maybe using a compose file?(or script).

**Reply by rickywu** [reply]
The same, maybe caused by registry mirror config, I update and use azuer as mirror then works fine
